faster_whisper/transcribe.py

```diff
@@ -218,77 +198,61 @@ def get_language_and_tokenizer(self, audio, task=None, language=None):
         return language, language_probability, task
     
     def audio_split(self, audio, segments, sampling_rate):
+        "Returns splitted audio chunks as iterator"
         for seg in segments:
             f1 = int(seg['start'] * sampling_rate)
             f2 = int(seg['end'] * sampling_rate)
             yield {'inputs': audio[f1:f2]}
 
-    def transcribe_stream(
-        self, audio: Union[str, np.ndarray], vad_segments, batch_size=None, num_workers=0, language=None, task=None, log_progress = False, combined_progress=False
-    ) -> TranscriptionResult:
+    def transcribe(
+        self, audio: Union[str, np.ndarray], vad_segments: List[dict], batch_size=16, num_workers=0, language=None, task=None, log_progress = False
+    ) ->Tuple[Iterable[BatchedSegment], dict]: 
```

Comment:
What's in the dict? Is there a TypedDict defined for it?

Reply:
As you can see in the yield line, it is a dictionary containing language and its probability. There is no TypedDict defined only for it.

Reply:
Uses TranscriptionInfo object instead of the dict.